src/status_im/signing/core.cljs

```diff
@@ -195,7 +195,9 @@
                    :signing/sign {:type           (cond pinless? :pinless
                                                         keycard-multiaccount? :keycard
                                                         :else :password)
-                                  :formatted-data (if typed? (types/json->clj data) (ethereum/hex-to-utf8 data))
+                                  :formatted-data (if typed? (types/json->clj data) (if (= 66 (count data))
```

Comment:
@richard-ramos can you please extract `66` to some const and give it a name? We could also extract the whole `if` form to a separate fn

Reply:
and please write some comments in the code why 66

Reply:
Done!